dsb-integration/src/test/java/com.spectralogic.dsbrowser.integration/ParseJobInterruptionMapTest.java

```diff
@@ -63,18 +71,32 @@ public static void setUp() throws Exception {
         Platform.runLater(() -> {
             try {
                 //Initiating session
-                final SavedSession savedSession = new SavedSession(SessionConstants.SESSION_NAME, SessionConstants.SESSION_PATH, SessionConstants.PORT_NO, null, new SavedCredentials(SessionConstants.ACCESS_ID, SessionConstants.SECRET_KEY), false);
-                session = new CreateConnectionTask().createConnection(SessionModelService.setSessionModel(savedSession, false));
+                final SavedSession savedSession = new SavedSession(
+                        "Ds3PutJobTest",
+                        client.getConnectionDetails().getEndpoint(),
+                        "80",
+                        null,
+                        new SavedCredentials(
+                                client.getConnectionDetails().getCredentials().getClientId(),
+                                client.getConnectionDetails().getCredentials().getKey()),
+                        false,
+                        false);
+                session = new CreateConnectionTask().createConnection(SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
+
                 //Initializing endpoint
                 endpoint = session.getEndpoint() + StringConstants.COLON + session.getPortNo();
+
                 //Loading resource file
-                final ClassLoader classLoader = ParseJobInterruptionMapTest.class.getClassLoader();
-                final URL url = classLoader.getResource(SessionConstants.LOCAL_FOLDER + SessionConstants.LOCAL_FILE);
-                if (url != null) {
-                    ParseJobInterruptionMapTest.file = new File(url.getFile());
+                final Path path;
+                try {
+                    path = ResourceUtils.loadFileResource("files/SampleFiles.txt");
+                    ParseJobInterruptionMapTest.file = path.toFile();
+                } catch (URISyntaxException | FileNotFoundException e) {
+                    e.printStackTrace();
```

Comment:
If this is a legitimate error condition, then let it fall out to the junit runner instead of handling it here, and swallowing the exception.